HomePage

shirishgoyal opened the pull request:
MD layout for homepage and SCSS

![image](https://cloud.githubusercontent.com/assets/315012/9704546/1c4a91d0-54c9-11e5-9e60-a00c671e64dc.png)

dmorina commented:
we will return to these, atm low priority, thanks

dmorina commented on static/css/_card.scss:
```diff
@@ -0,0 +1,5 @@
+
+md-card{
+  outline:0;
+}
```
not necessary to have its own scss file, originally it has one dedicated scss in angular-material

dmorina commented:
resolve conflicts to continue with review

shirishgoyal commented:
Made some changes as per discussion with Karolina
Updated Font to Source Sans Pro from Roboto
Removed Table Shadow

Resolved height issues in dashboard, My projects views
All code merge conflicts resolved

aginzberg commented:
"How it works" link keeps you on login. I assume it should take you to the homepage?

Spacing is a bit funky on a smaller screen:
<img width="1278" alt="screen shot 2015-11-04 at 11 16 28 am" src="https://cloud.githubusercontent.com/assets/4563883/10949020/92a2a4f4-82e5-11e5-96f6-f6218f37f752.png">

aginzberg commented:
our profiles pictures should probably be fixed size

dmorina commented on static/css/_button.scss:
```diff
@@ -0,0 +1,13 @@
+button{
```
unless we are implementing new components we shouldn't have these scss files, it's a bit confusing, move the styles to base or elsewhere

dmorina commented on static/css/app.css:
```diff
@@ -289,7 +327,7 @@ md-data-table {
     color: rgba(0, 0, 0, 0.87); }
   md-data-table .md-data-table-row-cell {
-    color: rgba(0, 0, 0, 0.87);
+    color: #243058;
```
leave it black, considering the specs says it should be black, should be overriden somewhere else

dmorina commented on static/templates/authentication/login.html:
```diff
@@ -1,3 +1,12 @@
+        <h1 class="md-display-3" layout="column" layout-align="center"><img src="/static/images/logo.svg"> DAEMO</h1>
+    </div>
+    <div flex layout="row" layout-align="center">
+        <md-button class="md-raised md-primary" ng-href="/login">Sign In</md-button>
```
remove signin button on top if the form is active

dmorina commented on static/templates/authentication/register.html:
```diff
@@ -1,3 +1,12 @@
+    <div flex layout="row" layout-align="center">
+        <md-button class="md-raised md-primary" ng-href="/login">Sign In</md-button>
+        <md-button class="md-raised md-accent"  ng-href="/register">Register</md-button>
+    </div>
```
remove sign-up button if we are on /register

shirishgoyal commented:
Removed "Our Profiles" button for now.
Handled all changes as per feedback.

dmorina commented:
we will redesign this based on new Master design 
reOpen if it becomes relevant again
